### PR TITLE
Docs: Added order of layout to style guide

### LINKED ISSUE #1444: Style guide: define the contract layout
Follow up of #1394:

> where to place modifiers, events, state variables? Probably we need a new section for "contract layout".

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
### Checklist
- [x] All tests are passing
- [x] README / documentation was extended, if necessary
- [x] Used meaningful commit messages

### Description

Adds order of code layout to style guide, closes https://github.com/ethereum/solidity/issues/1444

### CODE REVIEW

File: docs/style-guide.rst [hunk 1]
```diff
@@ -874,6 +874,27 @@ No::
     x = y+z;
     x +=1;
 
+***************
+Order of Layout
+***************
+
+We recommend you layout contract elements in the following order:
```
Comment (chriseth) [inline docs/style-guide.rst]: The "we-form" is not used anywhere else on this page, I think it mostly uses imperative.

**Review by chriseth — CHANGES_REQUESTED** [review]
Change form.

**Comment by ChrisChinchilla** [comment]
OK @chriseth switched.

**Review by chriseth — DISMISSED** [review]

**Comment by chriseth** [comment]
Please squash.

File: docs/style-guide.rst [hunk 2]
```diff
@@ -874,6 +874,28 @@ No::
     x = y+z;
     x +=1;
 
+***************
+Order of Layout
+***************
+
+Layout contract elements in the following order:
+
+1. Import statements
```
Comment (axic) [inline docs/style-guide.rst:883]: Should have added pragmas as the first one here.

Comment (ChrisChinchilla) [reply docs/style-guide.rst:883]: OK @axic will handle that asap.

Comment (chriseth) [reply docs/style-guide.rst:883]: Axic already did it.

**Pull request merged**